iridium-core-server/src/main/java/software/iridium/api/controller/AuthorizationController.java

```diff
@@ -70,4 +70,10 @@ public AccessTokenResponse exchange(
     logger.info("finalizing authorization");
     return authorizationService.exchange(servletRequest, params);
   }
+
+  @PostMapping(value = "/oauth/token/refresh")
+  public AccessTokenResponse refreshToken(@RequestParam Map<String, String> params) {
+    logger.info("access token request through refresh token");
+    return authorizationService.refreshToken(params);
+  }
```

Comment:
Let's add a test for this method.

Reply:
done

Reply:
I don't see a test.  Am I missing something?